app/views/general/_log_in_bar.html.erb

```diff
@@ -1,4 +1,4 @@
-<li id="logged_in_bar" class="logged_in_bar account-link-menu-item">
+<li id="logged_in_bar" class="logged_in_bar account-link-menu-item <%= 'pro-account' if @user && @user.is_pro? %>">
```

Comment:
This class is unused.

Also, no idea _why_ this was added from the commit message.